abc.templates/base-workflows/contents/guardian-admin.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - name: 'Setup Guardian'
         uses: 'abcxyz/pkg/.github/actions/setup-binary@3eaafa17e8e4b078507bed004ad5c57627d0e486' # ratchet:abcxyz/pkg/.github/actions/setup-binary@main
         with:
-          download_url: 'https://github.com/abcxyz/guardian/releases/download/v${{ env.GUARDIAN_VERSION }}/guardian_${{ env.GUARDIAN_VERSION }}_linux_amd64.tar.gz'
+          download_url: 'https://github.com/abcxyz/guardian/releases/download/${{ env.GUARDIAN_VERSION }}/guardian_${{ env.GUARDIAN_VERSION }}_linux_amd64.tar.gz'
```

Comment:
I dont think this is gonna work...... here is an example download URL:

The url contains the v, the binary doesnt.

`https://github.com/abcxyz/guardian/releases/download/v1.0.1/guardian_1.0.1_darwin_amd64.tar.gz`

Reply:
Reverted original changes and instead removed the `v` prefix before doing string replacement.

I'm assuming we should only accept valid semver tags, so I didn't check if the `v` prefix exists or not

Reply:
Our tags will always have a leading "v" because that's what Go expects.

Reply:
once the the next version of abc cli is released we can add a rule to check that the git tag matches a semvar format if we want to

Reply:
I'm also working on updating the release process so this will be codified everywhere